ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/snap/RetryingGetAccountRangeFromPeerTask.java

```diff
@@ -30,6 +30,7 @@
 public class RetryingGetAccountRangeFromPeerTask
     extends AbstractRetryingPeerTask<AccountRangeMessage.AccountRangeData> {
 
+  public static final int MAX_RETRIES = 5;
```

Comment:
Does it also make sense to increase the max retries to 5 for the other snap tasks? i.e. `RetryingGetBytecodeFromPeerTask`, `RetryingGetStorageRangeFromPeerTask` and `RetryingGetTrieNodeFromPeerTask`